src/main/antlr4/Manifold.g4

```diff
@@ -12,8 +16,9 @@ WHITESPACE: [ \t\r\n]+ -> skip;
 
 INTEGER_VALUE: [0-9]+;
 BOOLEAN_VALUE: 'false' | 'true';
+TYPE_KEYWORD: 'Type';
 
-tupleTypeValueEntry: (IDENTIFIER ':')? typevalue (':' expression)?;
+tupleTypeValueEntry: (IDENTIFIER ':')? typevalue ('=' expression)?;
 tupleTypeValue: '(' tupleTypeValueEntry (',' tupleTypeValueEntry)* ')';
 
 tupleValueEntry: (IDENTIFIER ':')? expression;
```

Comment:
Additionally, I think this should be an `=` to be consistent. `:` delineates a type, `=` delineates a value.